### ISSUE TITLE
Compiling place_receiver on frontera

### ISSUE THREAD

**Issue by Thomas-Ulrich** [SeisSol/Meshing#44]
Any idea why I get:
```
ulrich@login3:/work2/09160/ulrich/frontera/Meshing/place_receivers/build$ cmake ..
-- The C compiler identification is Intel 19.1.0.20200306
-- The CXX compiler identification is Intel 19.1.0.20200306
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /opt/intel/compilers_and_libraries_2020.4.304/linux/mpi/intel64/bin/mpiicc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/intel/compilers_and_libraries_2020.4.304/linux/mpi/intel64/bin/mpiicpc - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Set build type to Release as none was supplied.
-- Found HDF5: /usr/lib64/libhdf5.so (found version "1.8.12") found components: C HL 
-- Configuring done
-- Generating done
-- Build files have been written to: /work2/09160/ulrich/frontera/Meshing/place_receivers/build
ulrich@login3:/work2/09160/ulrich/frontera/Meshing/place_receivers/build$ make -j 
[ 80%] Building CXX object CMakeFiles/place_receivers.dir/src/Geometry.cpp.o
[ 80%] Building CXX object CMakeFiles/place_receivers.dir/src/main.cpp.o
[ 80%] Building CXX object CMakeFiles/place_receivers.dir/src/KDTree.cpp.o
[ 80%] Building CXX object CMakeFiles/place_receivers.dir/src/Reader.cpp.o
In file included from /work2/09160/ulrich/frontera/Meshing/place_receivers/src/Reader.cpp(42):
/work2/09160/ulrich/frontera/Meshing/place_receivers/submodules/PUML/PUML.h(100): error: expected an identifier
      auto[offset, size] = offsetAndSize(rank);
          ^
```
modules:
```
Currently Loaded Modules:
  1) intel/19.1.1   3) git/2.24.1      5) cmake/3.24.2   7) hwloc/1.11.12   9) TACC           11) seissol-env-develop-intel-19.1.1.217-w2i565p
  2) impi/19.0.9    4) autotools/1.2   6) pmix/3.1.4     8) xalt/2.10.34   10) python3/3.9.2
```

**Comment by davschneller** [comment]
Yes, that's because this notation is not present in C++11; it only exists since C++17. But the project only enables C++11 (and originally PUML was written in C++11 as well). Setting the `CMAKE_CXX_STANDARD` variable in `CMakeLists.txt` to 17 should work. The same is should work for all other projects which depend on PUML.

EDIT: I'll send in a PR pretty soon.
EDIT2: PR's out. It's #45.

**Comment by davschneller** [comment]
Fixed by #45.
